client/src/modules/list/model/actions.js

```diff
@@ -517,3 +526,21 @@ export const changeType = (listId, listName, type) => dispatch =>
         data: listName
       });
     });
+
+export const leaveList = (listId, userId, cohortId, userName) => dispatch =>
+  patchData(`/api/lists/${listId}/leave-list`)
```

Comment:
do we need this `-list` postfix? we know from context that it is about list with specific id